sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs

```diff
@@ -1,5 +1,13 @@
 namespace Azure.Provisioning
 {
+    public enum BicepKind
```

Comment:
While we prefer `Kind` over `Type` for most enums, I think it would be better here.  Will we be using this for more than params though?  There's way more to bicep's type system than just these simple values though - should we give it a different name more like `ParameterType`?

Reply:
It is already being used for both parameters and outputs.

Reply:
How about BicepDataType?

Reply:
Using BicepType